src/web/styles/themes.scss

```diff
@@ -25,19 +25,23 @@
   --theme-info-icon: #b7c2ea;
   --theme-info-content: #035959;
```

Comment:
looks like we can delete these two vars. `--theme-info-icon` is overridden by your new one on line 33 and `--theme-info-content` is unused

Reply:
removed.